### PR TITLE
first commit of content-layout

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
##Summary
add content-layout

##Screenshot
captura de pantalla 2018-09-10 a la s 17 31 53

##Trello Card
https://trello.com/c/mpZXF7Q8/23-content-layout

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,7 +1,18 @@
  @import 'color';
 
+ .body{
+  margin: 30px;
+
+ }
+
+ .container-technologies{
```
Comment (mmamaniwolox) [inline scss/tech.scss]: space

File: scss/tech.scss [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+  margin: 0 0 0 60px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: margin-left?

File: scss/tech.scss [hunk 3]
```diff
@@ -1,7 +1,18 @@
  @import 'color';
 
+ .body{
+  margin: 30px;
+
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Delete this empty line

File: scss/tech.scss [hunk 4]
```diff
@@ -1,7 +1,18 @@
  @import 'color';
 
+ .body{
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Space before curly braces

File: scss/tech.scss [hunk 5]
```diff
@@ -1,7 +1,18 @@
...
+  margin: 30px;
+
+ }
+
+ .container-technologies{
+  margin: 0 0 0 60px;
+  display: flex;
+  flex-wrap: wrap;
+ }
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Here you have an extra space

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**